src/delimiter.js

```diff
@@ -318,6 +313,112 @@ const makeStackedDelim = function(delim, heightTotal, center, options, mode,
+ 7-23 7l-12-1-109-253C131.333 778 94.667 694 94 694c-10.667 8-22 16.667-34 26
+ -22 17.333-33.333 26-34 26L0 720l76-59 76-60zM890 0h399110v40H901z\'/></svg>`,
+    size2 : `<svg viewBox=\'0 0 400000 1800\' preserveAspectRatio=\'xMinYMin
+slice\'><path fill=\'currentColor\' d=\'M890 0h399110v40H902.084S818.667 308
+638 880s-277 876.333-289 913c-4.667 4.667-12.667 7-24 7h-12c-1.333-3.333-3.667
+-11.667-7-25-35.333-125.333-106.667-373.333-214-744-10 12-21 25-33 39l-32 39
+c-6-5.333-15-14-27-26l25-30c26.667-32.667 52-63 76-91l52-60 208 722c56-175.333
+ 126.333-397.333 211-666s153.833-488.167 207.5-658.5C833.167 129.167 864 32.667
+ 872 10c4-6.667 10-10 18-10zm0 0h399110v40H902z\'/></svg>`,
+    size3 : `<svg viewBox=\'0 0 400000 2400\' preserveAspectRatio=\'xMinYMin
+slice\'><path fill=\'currentColor\' d=\'M313 2398c-1.333-.667-38.5-172-111.5
+-514S91.667 1370.667 91 1370c0-2-10.667 14.333-32 49-4.667 7.333-9.833 15.667
+-15.5 25s-9.833 16-12.5 20l-5 7c-4-3.333-8.333-7.667-13-13l-13-13 76-122 77-121
+ 209 968c0-2 84.667-361.667 254-1079C785.333 373.667 870.667 13.333 872 10c4
+-6.667 10-10 18-10h399110v40H903.622S816.332 418.667 631 1206c-185.333 787.333
+-279.333 1182.333-282 1185-2 6-10 9-24 9-8 0-12-.667-12-2z
+M890 0h399110v40H903z\'/></svg>`,
+    size4 : `<svg viewBox=\'0 0 400000 3000\' preserveAspectRatio=\'xMinYMin
+slice\'><path fill=\'currentColor\' d=\'M362 2713C701.333 913.667 871.333 13
+872 11c3.333-7.333 9.333-11 18-11h399110v40H906.698S816.168 518 630.5 1506
+C444.833 2494 351 2989 349 2991c-2 6-10 9-24 9-8 0-12-.667-12-2s-5.333-32-16-92
+c-50.667-293.333-119.667-693.333-207-1200 0-1.333-5.333 8.667-16 30l-32 64-16
+33-26-26 76-153 77-151c.667.667 35.667 202 105 604 67.333 400.667 102 602.667
+104 606zM890 0h399110v40H906z\'/></svg>`,
+    tall : `l-4 4-4 4c-.667.667-2 1.5-4 2.5s-4.167 1.833-6.5 2.5-5.5 1-9.5 1h
+-12l-28-84c-16.667-52-96.667 -294.333-240-727l-212 -643 -85 170c-4-3.333-8.333
+-7.667-13 -13l-13-13l77-155 77-156c66 199.333 139 419.667 219 661 l218 661z
+M591 0H400000v40H631z\'/></svg>`,
+};
+
+const sqrtSpan = function(height, delim, options) {
+    // Create a span containing an SVG image of a sqrt symbol.
+    const span = buildCommon.makeSpan([], [], options);
+    let sizeMultiplier = options.sizeMultiplier;  // default
+
+    if (delim.type === "small") {
+        // Get an SVG that is derived from glyph U+221A in font KaTeX-Main.
+
+        // First, get the corrrect sizeMultiplier for the surd.
+        const refHeight = height / sizeMultiplier;
+        let newOptions = options.havingBaseStyle(Style.TEXT);
+
+        //In the font, the glyph is 1000 units tall. The font scale is 1:1000.
+        if (1 / newOptions.sizeMultiplier < refHeight) {
+            newOptions = options.havingBaseStyle(Style.SCRIPT);
+            if (1 / newOptions.sizeMultiplier < refHeight) {
+                newOptions = options.havingBaseStyle(Style.SCRIPTSCRIPT);
+            }
+        }
```

Comment:
BTW, the difference between my test and the one inside `traverseSequence` is that my test used `height / options.sizeMultiplier` instead of `height` as the reference for comparison.